app/DoctrineMigrations/Version20160803183748.php

```diff
@@ -8,19 +8,20 @@
 /**
  * Auto-generated Migration: Please modify to your needs!
  */
-class Version20160729173849 extends AbstractMigration
+class Version20160803183748 extends AbstractMigration
```

Comment:
Do not overwrite migrations. Better create new ones to get a feeling of how to handle a lot of migrations.